src/solve.jl

```diff
@@ -110,7 +110,27 @@ function solve(system::InitialValueProblem{<:HybridSystem, <:LazySet{N}},
     # - (discrete) location
     # - (set of) continuous-time reach sets
     # - number of previous jumps
-    waiting_list = [(1, ReachSet{LazySet{N}, N}(X0, zero(N), zero(N)), 0)]
+    inits = length(options[:init]) > 0 ? options[:init] : [(n, X0) for n in states(HS)]
+
+    waiting_list = []
```

Comment:
Better initialize it with the right type: `Vector{Tuple{Int, ReachSet{LazySet{N}, N}, Int}}()`